ports/servoshell/egl/app_state.rs

```diff
@@ -101,6 +108,10 @@ impl ServoDelegate for ServoShellServoDelegate {
     fn notify_error(&self, _servo: &Servo, error: ServoError) {
         error!("Saw Servo error: {error:?}!");
     }
+
+    fn notify_animating_changed(&self, animating: bool) {
+        self.animating_state_changed.set(animating);
```

Comment:
This looks surprising. Anytime we transition from animating to not animating it won't be noticed, right? I believe we want to set this to true every time.

Reply:
Oh, yes. That's definitely a mistake. I will fix that before landing. Thanks!